### PR TITLE
Issue866 sup fan multizone

### LINKED ISSUE #842: Branch to create Trim&Respond logic
This branch intends to implement Trim&Respond logic.

### LINKED ISSUE #866: Multizone VAV AHU Supply fan control
Sequence for multizone VAV AHU supply fan control (G36, Part5.N.1)

### PR DESCRIPTION
**Pull request by JayHuLBL**
The sequence defines supply fan control logic for multizone AHU, G36 Part 5.N.1.

This closes #842
This closes #866

### CODE REVIEW

File: Buildings/Controls/OBC/ASHRAE/G36/Atomic/VAVMultiZoneSupFan.mo [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+        Text(
+          extent={{52,78},{94,58}},
+          lineColor={0,0,127},
+          textString="ySupFan")}),
+  Documentation(info="<html>
+<p>
+This block implements supply fan control of multizone VAV AHU according to 
+ASHRAE guideline G36, PART5.N.1 (Supply fan control).
+</p>
+<p>a. Supply fan start/stop</p>
```
Comment (mwetter) [inline Buildings/Controls/OBC/ASHRAE/G36/Atomic/VAVMultiZoneSupFan.mo]: Use `<h4>` formatting here and below.

File: Buildings/Controls/OBC/ASHRAE/G36/Atomic/VAVMultiZoneSupFan.mo [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+<li>If the AHU does not serve dual duct boxes (<code>duaDucBox=true</code>) or the AHU
+does not have airflow measurement station (<code>airFloMeaSta=false</code>),
+totallize current airflow rate from VAV boxes to a software point.</li>
+</ul>
+<p>b. Static pressure setpoint reset</p>
+Static pressure setpoint shall be reset using Trim-respond logic using following
+parameters as a starting place:
+<table summary=\"summary\" border=\"1\">
+<tr><th> Variable </th> <th> Value </th> <th> Definition </th> </tr>
+<tr><td>Device</td><td>AHU Supply Fan</td> <td>Associated device</td></tr>
```
Comment (mwetter) [inline Buildings/Controls/OBC/ASHRAE/G36/Atomic/VAVMultiZoneSupFan.mo]: There is no need to use `<code>` in the middle column as this is not code.

File: Buildings/Controls/OBC/ASHRAE/G36/Atomic/VAVMultiZoneSupFan.mo [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+<tr><td><code>I</code></td><td><code>2</code></td><td>Number of ignored requests</td></tr>
+<tr><td><code>R</code></td><td>Zone static pressure reset requests</td><td>Number of requests</td></tr>
+<tr><td><code>SPtrim</code></td><td><code>-12 Pa (-0.05 inches)</code></td><td>Trim amount</td></tr>
+<tr><td><code>SPres</code></td><td><code>15 Pa (+0.06 inches)</code></td><td>Respond amount</td></tr>
+<tr><td><code>SPres_max</code></td><td><code>32 Pa (+0.13 inches)</code></td><td>Maximum response per time interval</td></tr>
+</table>
+<br/>
+<p>c. Static pressure control</p>
+Supply fan speed is controlled to maintain duct static pressure at setpoint 
+when the fan is proven on. Where the Zone groups served by the system are
```
Comment (mwetter) [inline Buildings/Controls/OBC/ASHRAE/G36/Atomic/VAVMultiZoneSupFan.mo]: There is something wrong with the sentence "Where the zone groups...." This is not English.

**Comment by JayHuLBL** [comment]
@mwetter Your comments have been addressed
- [x] updated text in info section
- [x] removed fix

**Comment by mwetter** [comment]
@JayHuLBL : 
- [ ] In `G36.Atomic.VAVMultiZoneSupFan`, I still don't understand "Where the Zone groups served by the system are small, provided multiple sets of gains that are used in the control loop as a function of a load indicator (such as supply fan airflow rate, the area of the Zone groups that are occupied, etc.). " -- There is a verb missing.

**Comment by mwetter** [comment]
This is ready to merge when all tests pass.

**Pull request merged**